<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cameraSettingsDialog</class>
 <widget class="QDialog" name="cameraSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../screencast.qrc">
        <normaloff>:/pictures/camera/camera.png</normaloff>:/pictures/camera/camera.png</iconset>
      </attribute>
      <attribute name="title">
       <string notr="true"/>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxCameraResolution"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widgetCameraWindowSize" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>12</number>
            </property>
            <item alignment="Qt::AlignLeft">
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>34</width>
                <height>34</height>
               </size>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="pixmap">
               <pixmap resource="../screencast.qrc">:/pictures/camera/window-resize.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_44">
              <item>
               <widget class="QLabel" name="labelCameraWindowSize">
                <property name="text">
                 <string notr="true">---x---</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>12</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>34</width>
             <height>34</height>
            </size>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="pixmap">
            <pixmap resource="../screencast.qrc">:/pictures/systray/magnification.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_65">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_41">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_camera_view_rectangle">
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="icon">
            <iconset resource="../screencast.qrc">
             <normaloff>:/pictures/camera/window-rectangle-off.png</normaloff>
             <normalon>:/pictures/camera/window-rectangle-on.png</normalon>:/pictures/camera/window-rectangle-off.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>34</width>
             <height>34</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_35">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_camera_view_ellipse">
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="icon">
            <iconset resource="../screencast.qrc">
             <normaloff>:/pictures/camera/window-ellipse-off.png</normaloff>
             <normalon>:/pictures/camera/window-ellipse-on.png</normalon>:/pictures/camera/window-ellipse-off.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>34</width>
             <height>34</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_36">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_camera_view_circle">
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="icon">
            <iconset resource="../screencast.qrc">
             <normaloff>:/pictures/camera/window-circle-off.png</normaloff>
             <normalon>:/pictures/camera/window-circle-on.png</normalon>:/pictures/camera/window-circle-off.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>34</width>
             <height>34</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_37">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QCheckBox" name="checkBoxCameraMirrorVertical">
           <property name="text">
            <string>Flip vertical</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxCameraMirrorHorizontal">
           <property name="text">
            <string>Flip horizontal</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QCheckBox" name="checkBoxCameraInvert">
           <property name="text">
            <string>Invert</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxCameraGray">
           <property name="text">
            <string>Gray</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxCameraMono">
           <property name="text">
            <string>Black-and-white</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_42">
         <item>
          <widget class="QCheckBox" name="checkBoxCameraWindowFrame">
           <property name="text">
            <string>Remove window frame</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSwitchToFullscreen">
         <property name="text">
          <string notr="true">Switch to Fullscreen</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../screencast.qrc"/>
 </resources>
 <connections/>
</ui>
